pkg/iss/iss.go

```diff
@@ -761,7 +783,6 @@ func (iss *ISS) advanceEpoch() (*events.EventList, error) {
 	// (startEpoch emits an event for the application making it transition to the new epoch).
 	eventsOut.PushBackList(iss.startEpoch(
 		newEpochNr,
-		iss.epoch.leaderPolicy.Reconfigure(maputil.GetSortedKeys(iss.memberships[0])),
```

Comment:
Berofe starting the epoch, the leader policy still needs to be updated ("reconfigured") somewhere. Either here, before calling `startEpoch`, or when the memberships are being updated, or somewhere else where it's meaningful.

Reply:
Done.